docs/pages/enroll-resources/server-access/guides/host-user-creation.mdx

```diff
@@ -235,6 +237,135 @@ not already exist are created on the host. The `nginxrestarter` user is added to
 the `ubuntu`, `nginx`, and `other` groups, as specified in the `host_groups`
 field.
 
+## Static host users
+
+In this section, you will configure Teleport to create local users independently
+of an SSH session.
+
+### Step 1/3. Configure RBAC
+
+#### Label your server
+
+1. Access the server you enrolled with Teleport before beginning this guide and
+   open the Teleport configuration file, which is `/etc/teleport.yaml` by
+   default.
```

Comment:
This made me question - do static host users also get created on servers that use the server info resource for labeling?

Reply:
Yes, any labels that the server heartbeats back to auth are available for static host users.